About-page/index.html

```diff
@@ -0,0 +1,192 @@
+        <br>Hosted by Arsenal Football Club</br>
+        <a href="https://www.Arsenal.com" target="_blank">www.Arsenal.com</a>
+      </h3>
+    </div>
+
+      <div class="about-logo">
+       <h2 class="mobile">
+           Football Tactics Global series <br> 2nd Edition Logo </br>
+        </h2>
+        <h2 class="desktop">
+          Football Tactics Global series 2nd Edition Logo
+       </h2>
+        <div class="line-parent">
+            <hr class="line"/>
+        </div>
+
+        <h3>
+            The Football tactics forum was created through an free online compettition in Qatar 1. Feb to 3.March
+        </h3>
+
+        <div class="logo">
+            <img src="./images/1118.jpg" alt="forum logo">
+        </div> 
+      </div>
+    <section class="past-forum-pictures">
+         <div class="previous-photos">
+            <h2 class="mobile-words">
+                See the past FTF conferences
+            </h2>
+
+            <h3 class="mobile-words2">
+                Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+            </h3>
+            <h2 class="desktop-words">
+              See the past FTF conferences
+           </h2>
+
+           <h3 class="desktop-words2">
+              Take a look at the previous photographs from the FTC Conference that took place in England and Spain.
+           </h3>
+
+            <div class="photographs photo">
+                
+                  <div class="photograph">
+                    <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="first-image">
+                
+                  </div>
+
+                 <div class="photograph">
+                    <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="second-image">
+                 </div>
+            </div>
+            <div class="photographs-photo-desktop">
+                
+              <div class="photograph-desktop">
+                <img src="./images/FOS-22-6.27-Turkish-Super-Lig-BeinSports.jpg" alt="conference-first-photo" class="desktop-image">
+            
+              </div>
+
+             <div class="photograph-desktop">
+                <img src="./images/FSifztXXIAACC_2.jpg" alt="conference-second-image" class="desktop-image">
+             </div>
+
+            </div>
+          </div>
+    </section>
+
+
+    <section class="sponsoring-partners">
+        <div class="sponsors">
+            <h3 class="sponsor-title">Partners</h3>
+            <div class="line-second">
+                <hr class="line2"/>
+            </div>
+        </div>
+
+        <div class="partners">
+            <div><h2 class="sponsor1">Google</h2></div>
+            <div><h2 class="sponsor2">FIFA</h2></div>
+               <div><h2 class="sponsor3">UEFA</h2></div>
+        </div>
+        <div class="english-sponsor"> 
+            <h2>English Premier League</h2>
+        </div>
+
+    </section>
```

Comment:
- [ ] Kindly implement the requested by the previous reviewer that should have an image, title, and content. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description in 3 rows.
### Your design
![youuuu](https://user-images.githubusercontent.com/81727988/176983861-189f5356-fc88-4a5b-b662-baaf16362abd.png)
### Orignal design
![figma](https://user-images.githubusercontent.com/81727988/176983868-f3c89a58-df87-459e-8eef-fa3e0c65ad26.png)